ocaml/xenopsd/xc/domain.ml

```diff
@@ -886,15 +886,18 @@ let numa_placement domid ~vcpus ~memory =
         Softaffinity.plan ~vm host nodea
     )
   in
-  match hint with
+  let xcext = get_handle () in
```

Comment:
There's another `get_handle ()` above, L868, might want to bring that out into the function header